### PR TITLE
Sync nanomq conf with emqx.

### PR DESCRIPTION
**Pull request by lee-emqx**

### CODE REVIEW

File: etc/nanomq.conf [hunk 1]
```diff
@@ -1,94 +1,95 @@
...
+	num_taskq_thread = 0
+	max_taskq_thread = 0
+	parallel = 0
+	property_size = 32
+	max_packet_size = 1024
+	client_max_packet_size = 1024
+	msq_len = 2048
+	qos_duration = 10
+	keepalive_backoff = 1250
+	allow_anonymous = true
```
Comment (wanghaEMQ) [inline etc/nanomq.conf:20]: numtaskqthread & maxtaskqthread & parallel parameters should not be here. Which could be put to somewhere like sys{}.

File: etc/nanomq.conf [hunk 2]
```diff
@@ -1,94 +1,95 @@
...
-     file="nanomq.log"
-     rotation {
-             size=10MB
-             count=5
-     }
+	to = [file, console]
+	level = info
+	dir = "/tmp"
+	file = "nanomq.log"
+	rotation {
```
Comment (wanghaEMQ) [inline etc/nanomq.conf:57]: limitation could be more readable

File: etc/nanomq.conf [hunk 3]
```diff
@@ -1,94 +1,95 @@
...
-     username=admin
-     password=public
-     auth_type=basic
-     jwt {
-             public.keyfile="/etc/certs/jwt/jwtRS256.key.pub"
-             private.keyfile="/etc/certs/jwt/jwtRS256.key"
-     }
+	enable = true
+	port = 8081
+	parallel = 32
```
Comment (wanghaEMQ) [inline etc/nanomq.conf:41]: Is the parallel too big?

File: etc/nanomq.conf (hunk 1 above)
Comment (lee-emqx) [reply etc/nanomq.conf:20]: That's a good idea!

**Review by alvin1221 — APPROVED** [review]

**Review by JaylinYu — APPROVED** [review]

**Pull request merged**